src/pyodide/python-entrypoint-helper.ts

```diff
@@ -297,6 +301,13 @@ function makeEntrypointProxyHandler(
         prop = 'on_' + prop;
       }
 
+      if (!legacyGlobalHandlers && prop === 'test' && SHOULD_SNAPSHOT_TO_DISK) {
+        return async function () {
+          await getPyodide();
+          console.log('Stored snapshot to disk; quitting without running test');
+        };
+      }
+
```

Comment:
Can we split this change to a separate PR and in that PR also remove `make_snapshot = False` from the sdk test?

Reply:
This change doesn't fix the SDK test. This change actually keeps the behaviour the same as the global handlers, see https://github.com/cloudflare/workerd/pull/4769/files#diff-40b129fdc6739e5cb21248252f769e7d0fb708b8db9066bbc77b61605c00095eR34

Reply:
Huh, you're right. It does fix it. But then doesn't this mean that we are hiding a bug here?

Reply:
But sure, I can separate this out into another PR